test/pytest.ini

```diff
@@ -0,0 +1,20 @@
+[pytest]
+asyncio_mode = auto
+
+log_cli = true
+log_format = %(asctime)s.%(msecs)03d %(levelname)s>  %(message)s
+log_date_format = %H:%M:%S
+
+markers =
+    slow: tests that take more than 30 seconds to run
+    replication_factor: replication factor for RandomTables
+testpaths = test
+norecursedirs = test/manual test/perf test/lib test/raft test/unit
+# Ignore warnings about HTTPS requests without certificate verification
+# (see issue #15287). Pytest breaks urllib3.disable_warnings() in conftest.py,
+# so we need to do this here.
+filterwarnings =
+    ignore::urllib3.exceptions.InsecureRequestWarning
+
+tmp_path_retention_count = 1
+tmp_path_retention_policy = failed
```

Comment:
Do we really use tmp_path in our tests? I couldn't find where...

Reply:
These two options were used in test/object_store. [tmp_path_retention_count](https://docs.pytest.org/en/stable/reference/reference.html#confval-tmp_path_retention_count) and [tmp_path_retention_policy](https://docs.pytest.org/en/stable/reference/reference.html#confval-tmp_path_retention_policy) are configuration parameters for pytest about rention policy of the temporary directory created by pytest.

Reply:
Ok. I won't continue arguing about this here, but I'm still not happy that options that only one particular test needs suddenly need to appear in some global test/pytest.ini. That's not great. But it's not a disaster either so I'll not insist.